javaScriptAJAX-programming/checkResponseStatus/sampleAJAX.js

```diff
@@ -0,0 +1,11 @@
+var xhr = new XMLHttpRequest();
+xhr.onreadystatechange = function() {
+  if(xhr.readyState === 4) {
+    if(xhr.status === 200) {
+      const s_bar = document.getElementById('sidebar');
+      s_bar.innerHTML = xhr.responseText;
```

Comment:
Renaming `s_bar` to `sidebar` makes the code more understandable and follows common naming conventions where variable names are descriptive.